save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/OrganizationView.kt

```diff
@@ -67,95 +77,345 @@ class OrganizationView : AbstractView<OrganizationProps, OrganizationViewState>(
     init {
         state.isUploading = false
         state.organization = Organization("", null, null, null)
+        state.selectedMenu = OrganizationMenuBar.INFO
+        state.projects = emptyArray()
     }
 
     override fun componentDidMount() {
         super.componentDidMount()
         scope.launch {
             val avatar = getAvatar()
-            val organizationNew = getOrganization(props.organizationName)
+            val organizationLoaded = getOrganization(props.organizationName)
+            val projectsLoaded = getProjectsForOrganization()
             setState {
                 image = avatar
-                organization = organizationNew
+                organization = organizationLoaded
+                projects = projectsLoaded
             }
         }
     }
 
     @Suppress("TOO_LONG_FUNCTION", "LongMethod", "MAGIC_NUMBER")
     override fun RBuilder.render() {
-        div("d-sm-flex align-items-center justify-content-center mb-4") {
-            h1("h3 mb-0 text-gray-800") {
-                +"${state.organization?.name}"
+        when (state.selectedMenu!!) {
+            OrganizationMenuBar.INFO -> renderInfo()
+            OrganizationMenuBar.TOOLS -> renderTools()
+            OrganizationMenuBar.SETTINGS -> renderSettings()
```

Comment:
Can it be easily changed not clickable for now? Instead of redirection into blank page?

Reply:
We can hide it, but Alexander is already started making Settings view, so no need for it

Reply:
ok